src/plugins/template/tests/functional/op_reference/multiply.cpp

```diff
@@ -115,15 +106,79 @@ std::vector<MultiplyParams> generateParamsForMultiplyFloat() {
     return params;
 }
 
+template <element::Type_t IN_ET>
+std::vector<MultiplyParams> generateParamsForMultiply() {
+    using T = typename element_type_traits<IN_ET>::value_type;
+
+    std::vector<MultiplyParams> params{MultiplyParams(
+        ov::PartialShape{3, 2, 1},
+        ov::PartialShape{1, 6},
+        IN_ET,
+        std::vector<T>{12, 24, 36, 48, 60, 72},
+        std::vector<T>{1, 2, 3, 4, 6, 1},
+        std::vector<T>{12, 24, 36,  48,  72,  12, 24, 48,  72,  96,  144, 24, 36, 72,  108, 144, 216, 36,
+                       48, 96, 144, 192, 288, 48, 60, 120, 180, 240, 360, 60, 72, 144, 216, 288, 432, 72})};
+
+    return params;
+}
+
+template <>
+std::vector<MultiplyParams> generateParamsForMultiply<element::Type_t::i8>() {
+    constexpr auto IN_ET = element::Type_t::i8;
+    using T = typename element_type_traits<IN_ET>::value_type;
+
+    std::vector<MultiplyParams> params{MultiplyParams(
+        ov::PartialShape{3, 2, 1},
+        ov::PartialShape{1, 6},
+        IN_ET,
+        std::vector<T>{12, 24, 36, 48, 60, 72},
+        std::vector<T>{1, 2, 3, 4, 6, 1},
+        std::vector<T>{12, 24, 36,   48,  72, 12, 24, 48,  72,  96,  -112, 24, 36, 72,   108, -112, -40, 36,
+                       48, 96, -112, -64, 32, 48, 60, 120, -76, -16, 104,  60, 72, -112, -40, 32,   -80, 72})};
+
+    return params;
+}
+
+template <>
+std::vector<MultiplyParams> generateParamsForMultiply<element::Type_t::u8>() {
+    constexpr auto IN_ET = element::Type_t::u8;
+    using T = typename element_type_traits<IN_ET>::value_type;
+
+    std::vector<MultiplyParams> params{MultiplyParams(
+        ov::PartialShape{3, 2, 1},
+        ov::PartialShape{1, 6},
+        IN_ET,
+        std::vector<T>{12, 24, 36, 48, 60, 72},
+        std::vector<T>{1, 2, 3, 4, 6, 1},
+        std::vector<T>{12, 24, 36,  48,  72, 12, 24, 48,  72,  96,  144, 24, 36, 72,  108, 144, 216, 36,
+                       48, 96, 144, 192, 32, 48, 60, 120, 180, 240, 104, 60, 72, 144, 216, 32,  176, 72})};
+
+    return params;
+}
```

Comment:
Unless OV specification declares certain behavior for overflow, it shouldn't be tested.

Reply:
Dixed